net_util/src/tap.rs

```diff
@@ -16,6 +16,8 @@ use libc;
 use net_gen;
 use sys_util::{ioctl_with_mut_ref, ioctl_with_ref, ioctl_with_val};
 
+const IFACE_NAME_MAX_LEN: usize = 16;
```

Comment:
I suggest adding a comment here to say that this value is taken from the Linux kernel: https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/if.h#L33

Reply:
Sure. I've also added a test that validates this constant is still up to date.